docs/panel/advanced/docker.mdx

```diff
@@ -0,0 +1,203 @@
+# Panel in Docker
+Pelican provides pre-built Docker images via GitHub Packages. `ghcr.io/pelican-dev/panel:latest` is the current latest release, and `ghcr.io/pelican-dev/panel:main` is built automatically from the current `main` branch. Deploying the panel in Docker is still a work in progress. While the plan is to make Docker the preferred installation method, we currently recommend the [standard deployment instructions](/docs/panel/getting-started)
+
+This guide requires the Docker Engine and Docker CLI. (Docker Compose has been included in the Docker CLI since v2. Docker Compose v1 is unsupported.) If you need to install them, follow the [official install instructions](https://docs.docker.com/compose/install/).
```

Comment:
Let's keep this consistent with the other page here: https://pelican.dev/docs/wings/install/#installing-docker

Reply:
I moved that section to its own guide page and linked to it from both, to avoid maintaining it twice.